configure.ac

```diff
@@ -29,19 +32,45 @@ AC_ARG_ENABLE(
   [signature-verification],
   [AS_HELP_STRING([--enable-signature-verification], [Enable signature check (disabled by default)])],
   [AC_DEFINE([SIGNATURES], 1, [Enable signature check as default])]
-
+    SIGVERIFICATION=${enableval}
 )
 
 if test "$enable_signature_verification" = "yes" ; then
 	AC_ARG_WITH([swupdcert],
 		[AS_HELP_STRING([--with-swupdcert=FILE], [swupd verification cert])],
-		[AC_DEFINE_UNQUOTED([SWUPDCERT], ["$withval"], [swupd verification cert])],
+		[AC_DEFINE_UNQUOTED([SWUPDCERT], ["$withval"], [swupd verification cert])]
+		  SWUPDCERT=${withval},
```

Comment:
The configure is a little inconsistent with shell variable reference syntax, but we should standardize on one style, for readability.

Since the line above uses "$withval", I would use that here too.